setup.cfg

```diff
@@ -24,7 +24,7 @@ packages = anyio_serial
 python_requires = >= 3.6.2
 zip_safe = True
 install_requires =
-    anyio >= 2
+    anyio >= 2, <3
```

Comment:
Given that anyio3 is right around the corner …

Reply:
Yeah, I'll update to anyio 3